src/domain/community/community/community.entity.ts

```diff
@@ -60,6 +61,12 @@ export class Community
   })
   applications?: IApplication[];
```

Comment:
This surely is a mistake, right? We shouldn't have reference to GraphQL in our entities. Import should be updated as well...
```suggestion
  applications?: Application[];
```

Reply:
Agreed, also legacy code that had a bug.